### DISCUSSION TITLE
Intune and Winget Integration

### DISCUSSION THREAD

**Discussion by Kickinwing96** [microsoft/winget-cli#2641] [Q&A] [answered]
Good morning,
I was watching this [Microsoft Technical Takeoff](https://www.youtube.com/watch?v=11ezQMwnR_4&ab_channel=WindowsITPro) and I was wondering where/when I could access the preview for this new feature? (Found at [9:45](https://youtu.be/11ezQMwnR_4?t=585))
Any info is appreciated!
Sincerely,
A very interested School District O365 Admin

**Comment by mdanish-kh** [comment, marked as answer]
- https://github.com/microsoft/winget-cli/issues/215#issuecomment-1172420406

Please see this comment.

**Reply by Kickinwing96** [reply]
Thanks for pointing that out, I reached out to that individual.
Thanks again!
